slices/libpython3.8-minimal.yaml

```diff
@@ -0,0 +1,101 @@
+package: libpython3.8-minimal
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libssl1.1_libs
+    contents:
+      /etc/python3.8/sitecustomize.py:
```

Comment:
if you leave this out, will it still work? I'm wondering if this should be in a separate config slice, cause it's not a library

Reply:
So actually the `/usr/lib/python3.8/sitecustomize.py` library file is a symlink to this file. The `sitecustomize` module is imported in the `site` module, as per https://docs.python.org/3/library/site.html. 

However, any import error is silently ignored, thus this file can be removed if you really want to. But `/usr/lib/python3.8/sitecustomize.py` is a library file itself which is a symlink to the `/etc/python3.8/sitecustomize.py` file, that's why I kept that in the same slice.

Let me know if you still want me to relocate the `sitecustomize` file in a `config` slice and make that a dependency of this `libs` slice.

Reply:
ah ok. if it's a symlink to a lib then nevermind ;) ty